WordPress/WordPressShareExtension/ShareExtensionSessionManager.swift

```diff
@@ -226,6 +227,70 @@ import WordPressFlux
             self.cleanupSessionAndTerminate()
         })
     }
+
+    private func api(token: String) -> WordPressComRestApi {
+        return WordPressComRestApi(oAuthToken: token,
+                                   userAgent: nil,
+                                   backgroundUploads: false,
+                                   backgroundSessionIdentifier: backgroundSessionIdentifier,
+                                   sharedContainerIdentifier: WPAppGroupName)
+    }
+
+    private func token() -> String? {
+        return ShareExtensionService.retrieveShareExtensionToken()
+    }
+
+    private func updateMedia(postID: Int64?, siteID: Int64, onComplete: CompletionBlock?) {
+        guard let postID = postID else {
+            return
+        }
+
+        guard let service = mediaService(siteID: siteID) else {
+            return
+        }
+
+        coreDataStack.saveContext()
```

Comment:
I think we can remove this `saveContext()` call because we are not mutating any CD objects at this point and  it's also called on Line 214 above ☝️